experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.settings.ts

```diff
@@ -66,7 +66,7 @@ export const settings: IComposeSettings<IRadioButtonType> = [
       }
     },
     // This still uses the innerCircle's 'hover' style instead of the 'focused' style when need be. I'm commenting out'
-    _precedence: ['disabled', 'hovered', 'focused', 'selected'],
```

Comment:
focused is mentioned in the comment above, but moreover; are we removing 'focused' as a state because it's blocking/breaking, or just because it doesn't work atm?  If it's benignly inactive, I would leave it in and just track the bug(s) that make it work again

Reply:
Right now, this specific code doesn't break anything, however, it doesn't do anything either because of a change I had to make in my last PR that deals with Pressable not doing exactly what I was hoping :). In addition, this will actually need to change because of the token work I did. I removed this code to clean up the file a bit. I do have a task open to fix the underlying problem in which I'll add this back in.